### PR TITLE
add sample certificates for Observability documentation

### PR DESCRIPTION
**Pull request by YianZong**
# Description
Added sample certificates for Observability documentation

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/488 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Manually created certificates from samples
image

### CODE REVIEW

**Review by forrestxia — DISMISSED** [review]

**Review by taohe1012 — APPROVED** [review]

File: samples/observability/custom-cert.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+metadata:
+  name: otel-collector
+  namespace: karavi
+spec:
+  secretName: otel-collector-tls
+  duration: 2160h # 90d
+  renewBefore: 360h # 15d
+  subject:
+    organizations:
+    - dellemc
```
Comment (rensyct) [inline samples/observability/custom-cert.yaml:62]: Please change this to dell

File: samples/observability/selfsigned-cert.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+metadata:
+  name: otel-collector
+  namespace: karavi
+spec:
+  secretName: otel-collector-tls
+  duration: 2160h # 90d
+  renewBefore: 360h # 15d
+  subject:
+    organizations:
+    - dellemc
```
Comment (rensyct) [inline samples/observability/selfsigned-cert.yaml:22]: S comment as above

File: samples/observability/selfsigned-cert.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+metadata:
+  name: karavi-topology
+  namespace: karavi
+spec:
+  secretName: karavi-topology-tls
+  duration: 2160h # 90d
+  renewBefore: 360h # 15d
+  subject:
+    organizations:
+    - dellemc
```
Comment (rensyct) [inline samples/observability/selfsigned-cert.yaml:52]: Same comment as above

File: samples/observability/custom-cert.yaml [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+metadata:
+  name: karavi-topology
+  namespace: karavi
+spec:
+  secretName: karavi-topology-tls
+  duration: 2160h # 90d
+  renewBefore: 360h # 15d
+  subject:
+    organizations:
+    - dellemc
```
Comment (rensyct) [inline samples/observability/custom-cert.yaml:92]: Same comment as above

Comment (YianZong) [reply samples/observability/custom-cert.yaml]: Thanks @rensyct ! Changed all 'dellemc' to 'dell'.

**Review by forrestxia — APPROVED** [review]

**Pull request merged**